assets/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@serlo-org/athene2-assets",
```

Comment:
Location for new assets compiled by webpack. Will be published as a scoped, public npm module